app/pdf_utils/build_references.py

```
@@ -0,0 +1,33 @@
+"""
+    This function helps handle references within generated PDFs
+"""
+
+# TODO WORK IN PROGRESS
+
+
+def build_references(reference_dict):
+    """
+    Processes a reference/citation item. Returns a text string
+    """
+    # temp try
+    try:
+        # print(reference_dict, "reference_dict used to generate bib reference")
+
+        reference = ""
+
+        for key, value in reference_dict.items():
+            if not key:
+                continue
+            #  `series` gets changed to `journal` since `series` isn't a field used in
+            # the `@article` citation type of `apacite`
+            if key == "series":
+                reference += f"journal={{{value}}},\n"
+                continue
+            if key == "authors":
+                reference += f"author={{{value}}},\n"
+                continue
+            reference += f"{key}={{{value}}},\n"
+    except Exception as e:
+        raise e
```

If you just want to raise the exception as its thrown, I don't think there's a need to catch it

---

I'm not using this function yet and I'm leaning toward taking it out unless I see that it's necessary later for formatting references. I'll leave it alone for now.